src/main/java/bio/terra/service/auth/iam/IamProviderInterface.java

```diff
@@ -217,4 +217,17 @@ String getPetToken(AuthenticatedUserRequest userReq, List<String> scopes)
    *     statuses
    */
   RepositoryStatusModelSystems samStatus();
+
+  /**
+   * @param accessToken valid oauth token for the account fetching the group email
+   * @param groupName name of Firecloud managed group
+   * @return the group's email address
+   */
+  String getGroupEmail(String accessToken, String groupName) throws InterruptedException;
```

Comment:
I see you figured out my intention in this other comment:

https://github.com/DataBiosphere/jade-data-repo/pull/1361/files/23f8189381f996f7b3acae0bd06007dd2eb88086#r999573134

But I will add top-level comments to the `IamService` calls to indicate that it's the TDR SA who is performing these calls!  I don't want to bake that into the interface since we're generating the SA token upstream.